content/hi/firewall.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Firewall (फ़ायरवॉल)
```

Comment:
We should have the Hindi transliteration first and the english term in parenthesis () here. e.g. like in [https://glossary.cncf.io/hi/continuous_integration/](https://glossary.cncf.io/hi/continuous_integration/)

Reply:
done